files/image_config/platform/rc.local

```diff
@@ -265,6 +265,18 @@ if [ -f $FIRST_BOOT_FILE ]; then
         mkdir -p /host/reboot-cause/platform
     fi
 
+    if [ -d /host/image-$SONIC_VERSION/platform/common ]; then
+        env_file="/usr/share/sonic/device/$platform/platform_env.conf"
+        [ -f $env_file ] && . $env_file
+
+        gpl_dpkg="/host/image-$SONIC_VERSION/platform/common/opennsl-modules-dnx*.deb"
```

Comment:
why do we need different kernel module for dnx?

Reply:
Yes that was the recommendation from broadcom, to have separate GPL modules due to usage of compile time flags in code for different platforms. I have asked them again if possible to have a superset and a common driver.